rom-elasticsearch.gemspec

```diff
@@ -18,7 +18,6 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'rom-core', '~> 5.2', '>= 5.2.4'
```

Comment:
this must be updated, not removed

Reply:
I thought rom-core is merged to rom in rom 6.0, isn't it?

Reply:
yeah but adapters do depend on rom
https://github.com/rom-rb/rom-sql/blob/master/rom-sql.gemspec#L29